app/questionnaire/placeholder_renderer.py

```diff
@@ -36,6 +37,8 @@ def __init__(
         self._response_metadata = response_metadata
         self._schema = schema
         self._location = location
+        self._block_on_routing_path = False
```

Comment:
Could we rename it to `self._needs_block_on_routing_path` (or similar) to enforce its boolean type?
Even if it's initialised to False, I find it can get confusing when the variable is used elsewhere.
i.e "is it a block or a flag"?

Reply:
Yeah I agree sounds clearer . Added 👍